website/content/docs/discovery/dns.mdx

```diff
@@ -96,6 +96,33 @@ pairs according to [RFC1464](https://www.ietf.org/rfc/rfc1464.txt).
 Alternatively, the TXT record will only include the node's metadata value when the
 node's metadata key starts with `rfc1035-`.
 
+## Alternative Domain
+
+Consul support setting an alternative domain, which is usefull during DNS migrations or if you want to respond to
+internal and external queries with different domains.
+
+If you are using [`alt_domain`](/docs/agent/options#alt_domain) parameter, Consul will respond  based on the presence of it in your queries.
```

Comment:
I liked your use case examples! I made some changes to order and wording. Let me know what you think.

```suggestion
By default, Consul responds to DNS queries only for its configured
[`domain`](/docs/agent/options#domain).

Some use cases require responding to queries for more than one domain,
such as during a DNS migration or to distinguish between internal and
external queries by using different domains.

Consul can configured to respond to DNS queries on an alternative domain
through the [`alt_domain`](/docs/agent/options#alt_domain) agent configuration
option. Consul's DNS response will use the same domain as was used in the query.
```